static/templates/about.html

```diff
@@ -1,12 +1,12 @@
 {{template "main" .}}
 {{define "css"}}
-<link rel="stylesheet" href="static/stylesheets/about.css">{{end}}
+<link rel="stylesheet" href="/static/stylesheets/about.css">{{end}}
```

Comment:
I had to fix the href values for static resources to include the leading `/` because once you ended up at a path like `/about/fake` it was trying to load the static resources at `/about/static` instead of `/static`. Adding the leading `/` makes it specific so it can find the resources at the correct path.